src/navigators/TabNavigator.tsx

```diff
@@ -11,22 +11,34 @@ export type TabParamList = {
   SettingsTab: NavigatorScreenParams<SettingsStackParamList>;
 };
 
-const Tab = createBottomTabNavigator<TabParamList>();
+const Tab = createMaterialBottomTabNavigator<TabParamList>();
 
 export function TabNavigator() {
   return (
-    <Tab.Navigator>
+    <Tab.Navigator shifting={true}>
       <Tab.Screen
         name="HomeTab"
         component={HomeStack}
-        options={{ tabBarLabel: t('tabs-home', 'Home'), headerShown: false }}
+        options={{
+          tabBarLabel: t('tabs-home', 'Home'),
+          tabBarIcon: 'home',
+        }}
+      />
+      <Tab.Screen
```

Comment:
Love the way it looks but I think we should probably hide the notification tab until we have content for it.

Reply:
Roger that.